src/contribute/release-guide.md

```diff
@@ -158,12 +158,11 @@ Adjust any of the above properties to the improve clarity and presentation of th
 
 ### Verify that a Release Build Works
 
-Run `mvn -Prelease clean install` to ensure that the build processes that are specific to that
-profile are in good shape.
+Run `./gradlew -PisRelease clean build` to ensure that the build processes that are specific to that profile are in good shape.
```

Comment:
From a clean workspace `git clean -fdx`, perform `./gradlew -PisRelease build`

The gradle tasks you list don't run in the order that you specified, they just get added to the task tree and execute in dependency order (and if there is no dependency between them, then in arbitrary order).

Reply:
Added.

Reply:
I'm not sure whether we should instruct users to use `--no-parallel` or not here as well.

Reply:
ah yes that's true